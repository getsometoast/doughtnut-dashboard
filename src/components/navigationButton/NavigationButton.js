import React from 'react'
import FlatButton from 'material-ui/FlatButton'

const NavigationButton = ({label, changePage}) => (
  <FlatButton
    label='the label'
    onClick={changePage}
    primary={true}
    style={{
      width: '100%'
    }}
  />
);

export default NavigationButton
