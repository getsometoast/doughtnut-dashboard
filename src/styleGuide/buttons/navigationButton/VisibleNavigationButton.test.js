import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import NavigationButton from './VisibleNavigationButton';

describe('NavigationButton', () => {
  it('should render correctly', () => {
    const output = shallow(<NavigationButton />);
    expect(shallowToJson(output)).toMatchSnapshot();
  });
});
