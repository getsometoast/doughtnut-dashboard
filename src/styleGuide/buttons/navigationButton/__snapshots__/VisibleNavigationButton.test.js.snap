// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavigationButton should render correctly 1`] = `
<FlatButton
  disabled={false}
  fullWidth={false}
  label="go to about page via redux"
  labelPosition="after"
  labelStyle={Object {}}
  onClick={[Function]}
  onKeyboardFocus={[Function]}
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  onTouchStart={[Function]}
  primary={true}
  secondary={false}
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(observable): [Function],
    }
  }
  storeSubscription={
    Subscription {
      "listeners": Object {
        "clear": [Function],
        "get": [Function],
        "notify": [Function],
        "subscribe": [Function],
      },
      "onStateChange": [Function],
      "parentSub": undefined,
      "store": Object {
        "dispatch": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
        Symbol(observable): [Function],
      },
      "unsubscribe": [Function],
    }
  }
  style={
    Object {
      "width": "100%",
    }
  }
/>
`;
